priv/repo/migrations/20230712163553_rebuild_contained_pages.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Oli.Repo.Migrations.RebuildContainedPages do
+  use Ecto.Migration
+  import Ecto.Query, warn: false
+
+  alias Oli.Repo
+
+  def change do
+      from(s in "sections",
+      select: %{id: s.id, slug: s.slug, root_section_resource_id: s.root_section_resource_id}
+    )
+    |> Repo.all()
+    |> Enum.each(fn s ->
+      Oli.Delivery.Sections.rebuild_contained_pages(s)
```

Comment:
I wonder if we should copy+paste this `rebuild_contained_pages` implementation directly into this migration so that if we have to ever change this function in the future it doesn't affect this migration?

Reply:
I think in this case we will always want this migration to run whatever is the latest `rebuild_contained_pages`, regardless of when the migration is run.